interfaces/src/main/java/dotty/tools/dotc/interfaces/CompilerCallback.java

```diff
@@ -25,4 +27,12 @@
    *                        Example: ./src/library/scala/collection/Seq.scala
    */
   default void onSourceCompiled(SourceFile source) {};
+
+  default void sourceDependency(File dependsOn, File source, xsbti.DependencyContext context) {};
```

Comment:
I dont see what a method does given its name. Should it be `addSourceDependency`?